jsonschema/cli.py

```diff
@@ -178,6 +179,15 @@ def _namedAnyWithDefault(name):
         of the class.
     """,
 )
+parser.add_argument(
+    "--base-uri",
+    action="store_true",
```

Comment:
Thanks for changing this -- the idea here though was that this could/should be a URI, not just a flag.

I.e. you should say `--base-uri .` (note the `.`) if you mean "use the current directory as the base URI". But you could equally well pick any other directory as your base URI if you'd like as well.

Reply:
Ok, I have modified this part of the function. After the temporary file problem is modified, push together!